File: src/main/java/com/EmosewaPixel/pixellib/proxy/ServerProxy.java
package com.EmosewaPixel.pixellib.proxy;

import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.event.lifecycle.InterModEnqueueEvent;
import net.minecraftforge.fml.event.lifecycle.InterModProcessEvent;

@Mod.EventBusSubscriber(Dist.DEDICATED_SERVER)
public class ServerProxy implements IModProxy {
    @Override
    public void init() {

    }

    @Override
    public void enque(InterModEnqueueEvent e) {

    }

    @Override
    public void process(InterModProcessEvent e) {

    }
}

File: src/main/java/com/EmosewaPixel/pixellib/proxy/ServerProxy.kt
package com.EmosewaPixel.pixellib.proxy

import net.alexwells.kottle.KotlinEventBusSubscriber
import net.minecraftforge.api.distmarker.Dist
import net.minecraftforge.fml.event.lifecycle.InterModEnqueueEvent
import net.minecraftforge.fml.event.lifecycle.InterModProcessEvent

@KotlinEventBusSubscriber(Dist.DEDICATED_SERVER)
object ServerProxy : IModProxy {
    override fun init() {

    }

    override fun enque(e: InterModEnqueueEvent) {

    }

    override fun process(e: InterModProcessEvent) {

    }
}